package.json

```diff
@@ -1,18 +1,24 @@
 {
-  "name": "antd-theme-generator",
-  "version": "1.0.7",
-  "description":
-    "This a script to generate color specific styles less file and which you can use to change theme dynamically in browser",
+  "name": "antd-pro-theme-generator",
+  "version": "1.0.0",
+  "description": "This a script to generate color specific styles less file and which you can use to change theme dynamically in browser",
```

Comment:
`"This is a script to generate ....` error in orignal description, can you change it as part of your changes